src/modules/dashboard/components/SuggestionCreate/SuggestionCreate.tsx

```diff
@@ -26,54 +27,50 @@ interface FormValues {
   title: string;
 }
 
-interface InProps {
+interface Props extends RouteComponentProps {
   colonyAddress: Address;
   domainId: Domain['ethDomainId'];
 }
 
-interface Props extends InProps {
-  // Injected via `withDialog`
-  openDialog: OpenDialog;
-}
-
 const validationSchema = yup.object({
   title: yup.string().required(),
 });
 
 const displayName = 'Dashboard.SuggestionCreate';
 
-const SuggestionCreate = ({ colonyAddress, domainId, openDialog }: Props) => {
+const SuggestionCreate = ({ colonyAddress, domainId, history }: Props) => {
```

Comment:
I'm guessing you're changing that to use hooks as well :)

Reply:
Yep!